asyncio_mqtt/client.py

```diff
@@ -302,67 +387,91 @@ def _on_disconnect(self, client, userdata, rc, properties=None):
         else:
             self._disconnected.set_exception(MqttCodeError(rc, "Unexpected disconnect"))
 
-    def _on_subscribe(self, client, userdata, mid, granted_qos, properties=None):
+    def _on_subscribe(
+        self,
+        client: mqtt.Client,
+        userdata: Any,
+        mid: int,
+        granted_qos: int,
+        properties: mqtt.Properties = None,
```

Comment:
Let's use `Optional[...]` here as well. :)

Reply:
Oops, well spotted. Surprised that didn't give an error.